tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java

```diff
@@ -120,7 +120,7 @@ private File checkStaticServicesFile() throws IOException
                 }
             }
 
-            if( !supervisor.getAgent().redirect( -1 ) )
+            if( !supervisor.getAgent().redirect( Agent.COMMAND_SESSION ) )
```

Comment:
yeah , Greg , I just replace the int var with static var which is better to understand.